src/py/flwr/client/app.py

```diff
@@ -268,8 +269,16 @@ def start_numpy_client(
     >>>     root_certificates=Path("/crts/root.pem").read_bytes(),
     >>> )
     """
-    # Start
-    _check_actionable_client(client, client_fn)
+    warnings.warn(
+        "flwr.client.start_numpy_client() is deprecated and will "
+        "be removed in a future version of Flower.",
```

Comment:
Let's be more helpful here and add guidance on how to migrate from `start_numpy_client` to `start_client` using a multi-line warning that shows before/after.